CONTRIBUTING.md

```diff
@@ -62,7 +62,16 @@ included in the project:
    git checkout -b <topic-branch-name>
    ```
 
-4. Install the [Jekyll](https://github.com/jekyll/jekyll/) gem to preview locally.
+4. Install the [Jekyll](https://github.com/jekyll/jekyll/) gem and dependencies to preview locally:
+
+    ```bash
+    # Install the needed gems
+    gem install jekyll jekyll-sitemap pygments.rb
```

Comment:
@philsturgeon holy cow, that gem brought together a huge pile of gems! But that solved my trouble related on #639 as you said :)